### PR TITLE
docs(linux): Update 10.01 Issue Tracker

### PR DESCRIPTION
**Pull request by praneethbajjuri**
Issues Resolved and Open are going to be reflecting thru https://sir.ext.ti.com system.

This fix is to cleanup references from past release and add a small note clarifying the direction.

### CODE REVIEW

**Review by cshilwant — CHANGES_REQUESTED** [review]
Since its legacy platform,

s/10.1/9.3 in commit message

**Comment by praneethbajjuri** [comment]
Will send a followup PR with the Issue tracker section when all the new bugs resolved/closed are update in the sir system.

**Review by cshilwant — CHANGES_REQUESTED** [review]
For all note directives, code sections should clearly line up with directive
as mentioned in https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

File: source/devices/AM335X/linux/Release_Specific_Release_Notes.rst [hunk 1]
```diff
@@ -179,114 +179,11 @@ Yocto
...
-   "LCPD-9455","am335x-evm","Kernel Warning reported for a USB audio device when listing with pulseaudio",""
-   "LCPD-9372","am335x-evm","Nand stress tests failed on a particular am335x-evm board",""
-   "LCPD-8636","am335x-evm","Serial corruption being seen in kernel",""
-   "LCPD-8133","am335x-evm","USB: ""cannot reset"" errors observed sometimes",""
-   "LCPD-7955","am335x-evm","Uncorrectable Bitflip errors seen after switch to SystemD","Workaround to erase the NAND flash completely if flashed with an incompatible flash writer. SystemD tries to mount all partitions and that is the reason this is being seen now."
-   "LCPD-5362","am335x-evm","MUSB: Isoch IN only utilises 50% bandwidth",""
+.. note::
 
-|
+    - Release Specific Issues including details will be published thru Software Incident Report (SIR) portal
```
Comment (cshilwant) [inline source/devices/AM335X/linux/Release_Specific_Release_Notes.rst:184]:
```suggestion
    - Release Specific Issues including details will be published through Software Incident Report (SIR) portal
```
For all references of thru in PR,

https://www.grammarly.com/blog/commonly-confused-words/through-thru/#:~:text=Through%20is%20the%20only%20formally,%2Dthroughs%2Fdrive%2Dthrus.

File: source/devices/AM335X/linux/Release_Specific_Release_Notes.rst [hunk 2]
```diff
@@ -179,114 +179,11 @@ Yocto
...
-   "LCPD-14855","am335x-evm,am335x-ice,am335x-sk","omap_i2c_prepare_recovery() function can Lock System",""
-   "LCPD-13947","am335x-evm","nativesdk-opkg is broken in the devkit",""
-   "LCPD-10221","am335x-evm","Longer resume times observed on setup with usb device cable",""
-   "LCPD-9455","am335x-evm","Kernel Warning reported for a USB audio device when listing with pulseaudio",""
-   "LCPD-9372","am335x-evm","Nand stress tests failed on a particular am335x-evm board",""
-   "LCPD-8636","am335x-evm","Serial corruption being seen in kernel",""
-   "LCPD-8133","am335x-evm","USB: ""cannot reset"" errors observed sometimes",""
-   "LCPD-7955","am335x-evm","Uncorrectable Bitflip errors seen after switch to SystemD","Workaround to erase the NAND flash completely if flashed with an incompatible flash writer. SystemD tries to mount all partitions and that is the reason this is being seen now."
-   "LCPD-5362","am335x-evm","MUSB: Isoch IN only utilises 50% bandwidth",""
+.. note::
```
Comment (StaticRocket) [inline source/devices/AM335X/linux/Release_Specific_Release_Notes.rst:182]: I don't think this should be a note if it's the expected location for the release bug report moving forward. It should just be a paragraph about where they can find this information and why. For example:


```rst
To unify software issue tracking, release specific issue tracking is currently
handled through the `Software Incident Report (SIR) portal
<https://sir.ext.ti.com/>`.
```

**Review by StaticRocket — CHANGES_REQUESTED** [review]

File: source/devices/AM335X/linux/Release_Specific_Release_Notes.rst (hunk 2 above)
Comment (StaticRocket) [reply source/devices/AM335X/linux/Release_Specific_Release_Notes.rst:182]: Offline sync: This is not the permanent solution yet. We're fine with the note until a proper solution is determined.

**Review by StaticRocket — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Pull request merged**